cmake/FindRust.cmake

```diff
@@ -190,6 +194,7 @@ function(cargo_vendor)
     # for online builds when we run `cpack --config CPackSourceConfig.cmake`
     message(STATUS "Running `cargo vendor` to collect dependencies for ${ARGS_TARGET}. This may take a while if the local crates.io index needs to be updated ...")
     make_directory(${CMAKE_SOURCE_DIR}/.cargo)
+    GetCargoLocalDirEnv(_cargo_local_dir_env)
     execute_process(
         COMMAND ${CMAKE_COMMAND} -E env "CARGO_TARGET_DIR=${ARGS_BINARY_DIRECTORY}" ${cargo_EXECUTABLE} vendor "${CMAKE_SOURCE_DIR}/.cargo/vendor"
```

Comment:
Do we need to add `"${_cargo_local_dir_env}"` here?  Or is `GetCargoExtraEnv` not required for this function?

Reply:
See above (execute_process doesn't support lists)